### ISSUE TITLE
CRASH ! BOINC Manager 7.24.0 on macOS

### ISSUE THREAD

**Issue by Chamiu** [BOINC/boinc#5321]
**Describe the bug**
launch BOINC manager 7.24.0 on macOS Ventura.5 on MacBook Air Late 2020.

**Steps To Reproduce**
1. overwrite install from 7.22.2 to 7.24.0
2. launch 7.24.0

**Expected behavior**
normal launch.

**Screenshots**
attached error log

**System Information**
 - OS: macOS Ventura.5
 - BOINC Version: 7.24.0

**Additional context**
nothing
[BOINC 7.24.0 error.txt.zip](https://github.com/BOINC/boinc/files/12287106/BOINC.7.24.0.error.txt.zip)

**Comment by AenBleidd** [comment]
@CharlieFenton, could you please take a look?

**Comment by CharlieFenton** [comment]
I cannot reproduce this, installing BOINC 7.24.0 over 7.22.2 on my 2021 M1 iMac. It works correctly for me.

**Comment by CharlieFenton** [comment]
@Chamiu What language do you have BOINC and your Mac set for if other than English? Is there anything else about your setup that might explain why I can't reproduce the crash?

**Comment by Chamiu** [comment]
@CharlieFenton My Mac setting Preferred Language on System Preference.
1st. Japanese
2nd, English 
But user name is only ASCII and no include symbol.

I tried to set in use English to 7.24.0 installer.app to Application on System Preference.
But crash. 7.24.0.

I found this log in install.log in Console.app.
2023-08-08 12:43:51+09 (My Mac Name) package_script_service[26643]: ./postinstall: Tue Aug  8 12:43:51 2023   BOINC PostInstall error at line 955
